php/Terminus/Models/Environment.php

```diff
@@ -707,6 +654,26 @@ public function sendCommandViaSsh($command)
         return $response;
     }
 
+    /**
+     * Formats environment object into an associative array for output
+     *
+     * @return array Associative array of data for output
+     */
+    public function serialize()
+    {
+        $info = [
+          'id' => $this->id,
+          'created' => date(Config::get('date_format', $this->get('created'))),
+          'domain' => $this->domain(),
+          'onserverdev' => $this->get('on_server_development') ? 'true' : 'false',
```

Comment:
Yes, because they come out blank in the 1.x outputter. I'm amenable to using "yes" and "no", "true" and "false", or some other indicator instead.